admin/package-files/systemd/flocker-container-agent.service

```diff
@@ -4,7 +4,11 @@ After=docker.service
 Wants=docker.service
 
 [Service]
-ExecStart=/usr/sbin/flocker-container-agent --journald
+ExecStart=/usr/bin/sh -c '\
+    echo "Sleeping 120s to ensure previous dataset state is expired."; \
+    sleep 120; \
+    exec /usr/sbin/flocker-container-agent --journald \
```

Comment:
Since this is only a short-term work-around, can you link to the issue for a high-quality fix?  I think that issue should also be marked as a blocker for FLOC-3148.  And it wouldn't be terrible to have a link to that issue from here and the upstart file (since this isn't a trivial or incidental cleanup that's required, it's kind of a big deal).